### PR TITLE
[Core] Separate python wheel for each application

### PR DESCRIPTION
**Pull request by HubertBalcerzak**
Generating python wheels now produces a wheel for each application defined in `scripts/wheels/linux/applications` and `scripts/wheels/windows/applications`

Details of each application, like author or description, should be probably updated.

### CODE REVIEW

File: scripts/wheels/windows/applications/ContactStructuralMechanicsApplication.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+	"wheel_name": "KratosContactStructuralMechanicsApplication",
+	"included_modules": ["ContactStructuralMechanicsApplication"],
+	"included_binaries": ["KratosContactStructuralMechanicsApplication.pyd", "KratosContactStructuralMechanicsCore.dll"],
+	"dependencies": ["KratosMultiphysics==${KRATOS_VERSION}"],
+	"author": "Kratos Team",
```
Comment (loumalouomega) [inline scripts/wheels/windows/applications/ContactStructuralMechanicsApplication.json]:
```suggestion
	"author": "Vicente Mataix Ferrándiz ",
```

File: scripts/wheels/windows/applications/ContactStructuralMechanicsApplication.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+	"author_email": "kratos@listas.cimne.upc.edu",
```
Comment (loumalouomega) [inline scripts/wheels/windows/applications/ContactStructuralMechanicsApplication.json]:
```suggestion
	"author_email": "vmataix@cimne.upc.edu",
```

**Comment by loumalouomega** [comment]
We should think about including dependencies (ex. numpy, matplotlib) with this https://python-packaging.readthedocs.io/en/latest/dependencies.html

**Comment by HubertBalcerzak** [comment]
> 
> 
> We should think about including dependencies (ex. numpy, matplotlib) with this https://python-packaging.readthedocs.io/en/latest/dependencies.html

In this setup we can easly include dependencies by just adding them to the "dependencies" array in application (or kratos core) json.

**Review by loumalouomega — APPROVED** [review]
Thanks

**Pull request merged**